dask.Dockerfile

```diff
@@ -17,9 +17,9 @@ RUN conda install -c gpuci gpuci-tools
 
 RUN gpuci_conda_retry install -c conda-forge mamba
 
-RUN mamba env create -n dask --file /dask_environment.yaml
+RUN gpuci_mamba_retry env create -n dask --file /dask_environment.yaml
 
-RUN mamba install -y -n dask -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
+RUN gpuci_mamba_retry install -y -n dask -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge \
```

Comment:
Do we want to add `dask-cuda` here too (xref https://github.com/dask/dask/pull/7966#pullrequestreview-719431152)? I'm not sure if the latest `dask-cuda` release is what we'd like, or if we need the latest dev version. cc @pentschev @charlesbluca

Reply:
From the error you described, it sounds like the package we would want here is `dask_cudf`, I'm assuming whatever version corresponds with the version of `cudf` we have in the image.

Reply:
Thank @charlesbluca, my mistake, I meant `dask-cudf`

> I'm assuming whatever version corresponds with the version of cudf we have in the image.

Alright, in that case I think we'd want to add 

```
dask-cudf=$RAPIDS_VER \
```

here then

Reply:
Done

Reply:
Does `distributed` also need `dask-cudf`?

Reply:
I don't think it needs it right now.  But we mihgt start using that in the future.  At moment we mostly test cudf's serializers.  cc @pentschev